webpack/assets/javascripts/react_app/components/common/emptyState/index.js

```diff
@@ -0,0 +1,107 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { EmptyState as PfEmptyState, Button } from 'patternfly-react';
+
+const secondaryActionButtons = (actions = []) =>
+  actions.map(({ title, url }) => (
+    <Button key={`sec-button-${title}`} href={url}>
+      {title}
+    </Button>
+  ));
+
+const primaryActionButton = ({ title, url }) => (
+  <Button href={url} bsStyle="primary" bsSize="large">
+    {title}
+  </Button>
+);
+
+const documentationBlock = ({
```

Comment:
see https://github.com/theforeman/foreman/blob/develop/webpack/assets/javascripts/react_app/components/common/DocumentationLink/index.js, maybe worth refactoring / extracting

Reply:
Thanks @ohadlevy.
I uploaded [here](https://rawgit.com/ripcurld/foreman/emptystate_foreman/index.html?knob-primary%20action%20title=Primary&knob-icons=add-circle-o&knob-header=fjhsk&knob-description=ninjboy&knob-secondary%20action%20url=%23&knob-is%20switcher%20open=true&knob-current%20page=2&knob-secondary%20action%20title=Secondary&knob-total%20pages=3&knob-primary%20action%20url=%23&selectedKind=Empty%20State%20Pattern&selectedStory=with%20DocumentationLink&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybooks%2Fstorybook-addon-knobs) the storybook with EmptyState and DocumentationLink.

What bothers me is that DocumentationLink has the bullet in the beginning of the link. Is that supposed to be there?

Reply:
it is used today in the bookmarks dropdown, so I assume that needs to be extracted from the component and wrapped above it in bookmarks...?

Reply:
Yeah, I think if I move it from the `<MenuItem>` it will look good. Let me check it out

Reply:
Ping @ohadlevy - [here](https://rawgit.com/ripcurld/foreman/emptystate_foreman/index.html?knob-is%20switcher%20open=true&knob-total%20pages=3&knob-current%20page=2&knob-icons=key&knob-header=HEADER&knob-description=DESCRIPTION%21&selectedKind=Empty%20State%20Pattern&selectedStory=with%20Documentation%20Link%20Href&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybook%2Factions%2Factions-panel) is the storybook with what you suggested. I will upload the changes if it looks fine to you